tock/hours/models.py

```diff
@@ -322,6 +322,14 @@ def calculate_target_hours(self):
         total_hours_worked = settings.HOURS_IN_A_REGULAR_WORK_WEEK - self.excluded_hours
         return round(self.billable_expectation * total_hours_worked, 0)
 
+    def calculate_total_allocation_hours(self):
+        """
+        Calculates an estimate of the hours spent on projects with a weekly billable
```

Comment:
Non-blocking question: Is this just an "estimate" because the allocation percentage is itself not a precise measurement? I.e., allocation represents how a user has been staffed and not necessarily an exact measurement of billable hours?

Reply:
My understanding is that a person with 100% allocation on a project could be working anywhere between 32 and 40 hours, but it's not possible for us to know exactly what.  When we talked to Matt S the other day, I think he said he's using 34 (or was it 36?) hours when utilization is 100%.

Reply:
Got it, thanks for clarifying!

I don't have any suggestions for changes to this PR, but I think _I_ should include a callout in my documentation updates (#1611). Something under "billable allocation hours" that notes the intent and limitations of the derived value. I.e., "We need a number so we can perform utilization rate calculations across dimensions that include varying combinations of hourly, weekly, and excluded hours projects. The powers that be have decided x, y, z for translating allocation to hours for this purpose."

Oh, and yeah, my notes say Matt S treats 100% as 36.